src/rust/rust-proto-new/tests/test_utils/strategies.rs

```diff
@@ -0,0 +1,23 @@
+use bytes::Bytes;
```

Comment:
We can totally add more strategies here, I'm mostly doing this PR as a proof-of-concept / "hey here's how you'd add more stuff without siloing it to one file"